hadoop-hdds/docs/content/security/SecuringDatanodes.md

```diff
@@ -38,7 +38,7 @@ that is setup in  hdfs-site.xml.
 Property|Description
 --------|--------------
 dfs.datanode.kerberos.principal|The datanode service principal. <br/> e.g. dn/_HOST@REALM.COM
-dfs.datanode.keytab.file| The keytab file used by datanode daemon to login as its service principal.
+dfs.datanode.kerberos.keytab.file| The keytab file used by datanode daemon to login as its service principal.
```

Comment:
Can you please point out where `dfs.datanode.kerberos.keytab.file` is used?

`dfs.datanode.keytab.file` is defined in https://github.com/apache/ozone/blob/d6cbb15f7a29221fcbd8f2c3bc9f5073c12c12d7/hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/DFSConfigKeysLegacy.java#L72-L73 and used in https://github.com/apache/ozone/blob/d6cbb15f7a29221fcbd8f2c3bc9f5073c12c12d7/hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/HddsDatanodeService.java#L244-L254

Reply:
Thanks to @adoroszlai ’s review, I think the original intention of this [jira](https://issues.apache.org/jira/projects/HDDS/issues/HDDS-6119) is to change `dfs.datanode.keytab.file` to `dfs.datanode.kerberos.keytab.file`, which is similar to naming conventions such as `dfs.datanode.kerberos.principal`. 
The purpose of this issue is to change its naming for consistency. Do you think this is feasible? If change the naming, the code you mentioned also needs to be changed.

Reply:
Thanks for the explanation.

I guess we can change the property name, but need to keep backwards compatibility by adding deprecation.

Doc change in itself is not OK, as it introduces mismatch.